src/main/java/net/ossindex/version/InvalidRangeRuntimeException.java

```diff
@@ -0,0 +1,17 @@
+package net.ossindex.version;
+
+/**
+ * The parser is not happy with us injection a regular exception, so make this one a RuntimeException. We will
```

Comment:
I don't understand `with us injection`.  And here's a random tomato: 🍅

Reply:
lol

s/injection/injecting/